src/tcgwars/logic/impl/gen4/RisingRivals.groovy

```diff
@@ -2605,24 +3491,49 @@ public enum RisingRivals implements LogicCardInfo {
           weakness D, PLUS20
           resistance C, MINUS20
           pokePower "Mow Shift", {
-            text "Once during your turn , you may use this power. Mow Rotom’s type is until the end of your turn."
+            text "Once during your turn , you may use this power. Mow Rotom’s type is [G] until the end of your turn."
             actionA {
+              checkLastTurn()
+              powerUsed()
+              bc "$self's type is now GRASS until the end of this turn."
+              delayed {
+                def eff
+                register {
+                  eff = getter GET_POKEMON_TYPE, self, { h->
+                    h.object.clear()
+                    h.object.add(G)
+                  }
+                }
+                unregister {
+                  eff.unregister()
+                }
+                unregisterAfter 1
+              }
             }
           }
           move "Slash", {
             text "20 damage. "
             energyCost C, C
             attackRequirement {}
             onAttack {
-              damage 0
+              damage 20
             }
           }
           move "Mow Down", {
             text "50 damage. Flip a coin. If heads, discard an Energy card attached to each of your opponent’s Pokémon."
-            energyCost C, C
+            energyCost G, C, C
             attackRequirement {}
             onAttack {
-              damage 0
+              damage 50
+              flip {
+                afterDamage {
+                  opp.all.each {
+                    if(it.cards.filterByType(ENERGY)) {
+                      it.cards.select("Choose an Energy card to discard",cardTypeFilter(ENERGY)).discard()
+                    }
+                  }
```

Comment:
Make a new discardEnergyAfterDamage static if you need to, but this is incorrect behaviour I'm pretty sure.